public/js/actions/user.js

```diff
@@ -127,3 +127,25 @@ export function userSignOut(jquery=$) {
     });
   };
 }
+
+
+export function getBraintreeToken() {
+  console.log('Requesting braintree token');
+  return dispatch => {
+    $.ajax({
+      method: 'post',
+      url: '/api/braintree/token/generate/',
+      context: this,
+    }).then(data => {
+      dispatch({
+        type: actionTypes.RX_BRAINTREE_TOKEN,
+        user: {
+          braintreeToken: data.token,
+        },
```

Comment:
since this action only returns a token, I don't think you need to wrap it in a user object. Could just be:
````
dispatch({
  type: ...,
  braintreeToken: data.token,
})
````